.travis.yml

```diff
@@ -20,12 +20,20 @@ deploy:
   timeout: 60
   module_name: app
   handler_name: handle_event
-  environment_variables:
-  - LOG_LEVEL=debug
-  - PLATFORM_API_BASE_URL=https://dev-platform.nypl.org/api/v0.1
-  - SCHEMA_TYPE=Holding
-  - KINESIS_STREAM=HoldingPostRequest-dev
-  - LOCATIONS_JSONLD_URL=https://s3.amazonaws.com/nypl-core-objects-mapping-production/by_sierra_location.json
+  event:
+    event_source_arn: arn:aws:kinesis:us-east-1:224280085904:stream/SierraHoldingsParser-dev
+    batch_size: 10
+    maximum_record_age_in_seconds: 86400
+    bisect_batch_on_function_error: true
+    maximum_retry_attempts: 3
+    starting_position: LATEST
+  environment:
```

Comment:
This is interesting. This provides the env vars in the way that AWS expects them, right? But also this is *not* what travis expects, but it doesn't matter, because it doesn't work anyway. *And* I think the the next version of the travisCI integration (documented [here](https://docs.travis-ci.com/user/deployment-v2/providers/lambda/)) uses this.

So in short I think this is the right way to go, but want to make sure that I'm making the right assumptions in thinking that.

Reply:
Yes, this was based around the idea of making the config as close as possible to what aws expects. I was a little worried that this would be an issue for travis, but it doesn't seem to cause any problems.